packages/code-du-travail-data/dataset/preavis-licenciement/index.js

```diff
@@ -0,0 +1,113 @@
+const { promisify } = require("util");
+const GoogleSpreadsheets = require("google-spreadsheets");
+
+const SPREADSHEET_KEY = "1zd_hShEui8BHK0349GpDUZRkCcQ9syIZ9gSrkYKRdo0";
+const SPREADSHEET_TAB = 4;
+const SPREADSHEET_QUESTIONS_TAB = 6;
```

Comment:
this looks a complete copy / paste of script in preavis-demission (except for the filter), If it the same, I wonder if we should move it a function now or wait for another tool.

Reply:
Agree with you, maybe we can wait another tool before refactoring because we don't know actually how tools can evolve